ros_gz_shims/ros_ign_bridge/src/parameter_bridge_shim.cpp

```diff
@@ -0,0 +1,30 @@
+// Shim to redirect "ros_ign_bridge parameter_bridge" call to "ros_gz_bridge parameter_bridge"
+
+#include <string>
+#include <sstream>
+#include <iostream>
+#include <stdlib.h>
+
+#include <ament_index_cpp/get_package_prefix.hpp>
+
+
+int main(int argc, char * argv[])
+{
+  std::stringstream cli_call;
+
+  cli_call << ament_index_cpp::get_package_prefix("ros_gz_bridge")
+           << "/lib/ros_gz_bridge/parameter_bridge";
```

Comment:
I hope this assumption about the directory structure holds for all supported platforms. This works for me on a colcon workspace. 

The installed binaries end up in e.g. `/opt/ros/galactic/lib/ros_ign_bridge/parameter_bridge`. Are we sure that the package prefix would be `/opt/ros/galactic` in this case?

Reply:
The package prefix will be the install space of the top-most (overlay) workspace with this package inside.
So.. I think so?

Reply:
I did some GitHub searching, and it looks like that should work for debs. We could make this more robust for other platforms, but I'm not sure if the `system` call would work on Windows anyway

https://github.com/klintan/rosbag2.js/blob/65ebe15d242c06efb026c11f3ced35065f80e7d6/include/rosbag2_converter_default_plugins/cdr/cdr_converter.cpp#L35-L59